src/table.js

```diff
@@ -578,13 +578,56 @@ Table.prototype.delete = function(gaxOptions, callback) {
   );
 };
 
+/**
+ * Truncate the table.
+ *
+ * @param {object} [gaxOptions] Request configuration options, outlined
+ *     here: https://googleapis.github.io/gax-nodejs/CallSettings.html.
+ * @param {function} callback The callback function.
+ * @param {?error} callback.err An error returned while making this request.
+ * @param {object} callback.apiResponse The full API response.
+ *
+ * @example
+ * table.truncate(function(err, apiResponse) {});
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * table.truncate().then(function(data) {
+ *   var apiResponse = data[0];
+ * });
+ */
+Table.prototype.truncate = function(gaxOptions, callback) {
```

Comment:
This isn't my favorite request to make, but we try to keep our methods defined alphabetically so that it's easier to dig into the src/tests. Would you mind relocating it and the describe block in the tests?